src/Routing/RouteBuilder.php

```diff
@@ -612,7 +612,12 @@ public function prefix($name, $params = [], callable $callback = null)
             $params = [];
         }
         $name = Inflector::underscore($name);
-        $path = '/' . $name;
+        if (isset($params['path'])) {
+            $path = '/' . $params['path'];
```

Comment:
The slash should be added only if doesn't already exist. For e.g. my plugin already sets the path with starting slash https://github.com/ADmad/CakePHP-HybridAuth/blob/master/config/routes.php#L13